### ISSUE TITLE
Missing aria-hidden on Jorge

### ISSUE THREAD

**Issue by abbydrury** [mtholyoke/lits_theme#16]
The Jorge image on the front page in the Featured Resources should be marked as decorative in some way, like the FontAwesome icons. No cookie to Abby for forgetting this.
